tests/Api/Admin/PaymentMethodsTest.php

```diff
@@ -205,4 +205,31 @@ public function it_does_not_update_a_payment_method_with_duplicate_locale_transl
             Response::HTTP_UNPROCESSABLE_ENTITY,
         );
     }
+
+    /** @test */
+    public function it_gets_gateway_config(): void
+    {
+        $fixtures = $this->loadFixturesFromFiles([
+            'authentication/api_administrator.yaml',
+            'channel.yaml',
+            'payment_method.yaml',
+        ]);
+
+        $header = array_merge($this->logInAdminUser('api@example.com'), self::CONTENT_TYPE_HEADER);
+
+        /** @var PaymentMethodInterface $paymentMethod */
+        $paymentMethod = $fixtures['paypal_payment_method'];
+
+        $this->client->request(
+            method: 'GET',
+            uri: sprintf('/api/v2/admin/gateway-configs/%s', $paymentMethod->getGatewayConfig()->getId()),
```

Comment:
If it is `PaymentMethodsTest`, should it not be `admin/payment-methods/{code}/gateway-config`?